platforms/hyperledger-indy/configuration/roles/setup/vault_kubernetes/README.md

```diff
@@ -0,0 +1,117 @@
+## setup/vault_kubernetes
+This role check HashiCorp Vault for existing of policies, roles and auth method. When these don't exist, then creates them.
```

Comment:
Isn't this role just for the admin-vault-auth SA? I would emphasise it in this README.